sys/kern/kern_event.c

```diff
@@ -990,16 +990,14 @@ kern_kevent_generic(struct thread *td, struct g_kevent_args *uap,
 #ifdef KTRACE
 	if (KTRPOINT(td, KTR_GENIO)) {
 		kgio = ktr_geniosize;
-		ktriov.iov_base = uap->changelist;
-		ktriov.iov_len = kev_iovlen(uap->nchanges, kgio,
-		    k_ops->kevent_size);
+		IOVEC_INIT(&ktriov, uap->changelist,
```

Comment:
FYI, this has changed upstream and I think would just go away after the next merge since this now uses a new 'ktrstructarray()' function.